whitelabelco/sofo-foods
# Cp/bug/hero buttons mobile

### Description 💻

Resolves: #137 

### Implementation 💡

fixes gap between product hero and drop down on mobile view
fixes gap between nav and Brands hero
hides secondary button on mobile view on Products and Brands

Reference ticket for bug demo

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- switch to mobile view
- click on Products
- verify spacing between hero and drop down matches comps, verify secondary button is hidden (but displays on all larger screens)
- Click on Brands
- verify gap between nav and hero is gone, verify secondary button in hidden (but displays on all larger screens)

### Screenshots/Demos 📷
![Screenshot 2024-10-23 at 5 54 18 PM](https://github.com/user-attachments/assets/81b8d46b-c6d2-4db3-8fb9-9acf9607ded6)
![Screenshot 2024-10-23 at 5 54 30 PM](https://github.com/user-attachments/assets/629bfb3f-4849-41a9-ab25-322823085c74)





### Checklist ✅
